test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -163,6 +174,41 @@ protected virtual void CreateTestStorageEntities()
             ServiceBusQueueClient = Microsoft.ServiceBus.Messaging.QueueClient.CreateFromConnectionString(connectionString, serviceBusQueueName);
         }
 
+        private IProxyClient GetMockProxyClient()
+        {
+            var proxyClient = new Mock<IProxyClient>();
+
+            ProxyData proxyData = new ProxyData();
+            proxyData.Routes.Add(new Routes()
```

Comment:
Can you put some code comments in here so it's clearer what is going on with these routes?